liquibase-core/src/test/java/liquibase/changelog/visitor/DBDocVisitorTest.java

```diff
@@ -0,0 +1,90 @@
+package liquibase.changelog.visitor;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+import org.mockito.MockedStatic;
+
+import liquibase.CatalogAndSchema;
+import liquibase.database.Database;
+import liquibase.database.core.OracleDatabase;
+import liquibase.database.core.PostgresDatabase;
```

Comment:
I do not believe the import for liquibase.database.core.PostgresDatabase is needed for this test.